packages/gatsby/src/utils/eslint-rules/limited-exports-page-templates.ts

```diff
@@ -0,0 +1,80 @@
+import { Rule } from "eslint"
+import { Node, Identifier } from "estree"
+import { store } from "../../redux"
+import { isPageTemplate } from "../eslint-rules-helpers"
+
+function isTemplateQuery(node: Node, varName: string | undefined): boolean {
+  // Checks for:
+  // const query = graphql``
+  // export { query }
+  if (
+    node?.type === `ExportNamedDeclaration` &&
+    node?.declaration === null &&
```

Comment:
Is `Node` defined as nullable? It's not in code so unless TS requires the `?.` those could be dropped.